app/views/notes/_note.html.erb

```diff
@@ -1 +1 @@
-<article data-width="<%= note.width %>" data-height="<%= note.height %>" data-x="<%= note.x %>" data-y="<%= note.y %>" data-id="<%= note.id %>"><%= raw DText.sanitize(note.body) %></article>
+<article data-width="<%= note.width %>" data-height="<%= note.height %>" data-x="<%= note.x %>" data-y="<%= note.y %>" data-id="<%= note.id %>" data-body="<%= note.body %>"><%= raw DText.sanitize(note.body) %></article>
```

Comment:
I tested it and as far as I can tell it works. Rails escapes `"lol"` as `&quot;lol&quot;` for the data-body attribute, but the browser unescapes that back to `"lol"` so it still works.